RFCs/RFC-0: Lightweight RFC process.md

```diff
@@ -0,0 +1,69 @@
+# RFC-0: Lightweight RFC Process for riff
+
+**To be Reviewed By:** Nov 15th 2019
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Dropped | Superseded
+
+**Superseded by:** N/A
+
+**Related:** N/A
+
+
+## Problem
+As the scope of the project expands and the number of moving pieces increase, we need a process to gather feedback on the changes proposed to the architecture and interaction of various components in the system.
+
+This document will refer to that process as the “Request For Comments (RFC) process”.
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or a group of authors submit a proposal to the community in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
+
+Much inspiration for this proposal has been drawn from [Apache Geode's RFC process](https://cwiki.apache.org/confluence/display/GEODE/Lightweight+RFC+Process), which inturn draws from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html).
+
+All RFCs are submitted via PRs which will be merged when approved.
+
+### Collaboration
+Comments and feedback should be provided on the PR and focus on the technical content.
+
+Authors should address all comments written by the deadline. This doesn't mean every comment and suggestion must be accepted and incorporated, but they should be carefully read and responded to.
+
+Every RFC has a lifecycle. The life cycle has the following phases:
+* **Active**: The RFC has been drafted and is open for comments. When making the PR, the doc should start out in this state.
+* **Accepted**: All the comments have been addressed and the changes proposed on this RFC have been accepted. This should be the state while merging the PR.
+* **Dropped**: The changes proposed on this RFC aren't in effect anymore, the document is kept for historical purposes and there is a new RFC that’s more current.
```

Comment:
Should the PR be closed?

Reply:
If replaced by a "more current" RFC, how is this different from Superseded? And why is Superseded not listed as a "phase"?

Reply:
Changed Dropped and added Superceded:
```
* **Dropped**: The changes proposed on this RFC were not agreed upon and no implementation will follow.
* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document may be kept for historical purposes and there is a new RFC that’s more current.
```